INSTALLATION.md

```diff
@@ -1,76 +1,78 @@
+
 # Installation
 
 * [Introduction](#introduction)
-* [Step by step instructions](#step-by-step-instructions)
+* [Desktop](#desktop)
   * [MacOS](#macos)
   * [Linux](#linux)
   * [Windows](#windows)
+* [Mobile](#mobile)
 
 ## Introduction
 
-This document will guide you through the installation process of the latest tagged version of Thyra.
-
-> **_PREREQUISITES:_** Be comfortable with your terminal and have a recent version of MacOS, Windows or Linux.
-
-> **_TROUBLESHOOTING:_** If you have trouble following this procedure, feel free to [open a question](https://github.com/massalabs/thyra/issues/new) describing your problem.
-
-## Step by step instructions
+From this point you'll start to custom Massa's journey thanks to Thyra. If you want to learn more about Thyra's objectives and mindset, go here; if you're curious about the technical aspects and want to contribute go [there](./CONTRIBUTING.md).
 
-Two steps are required to use our web on-chain product:
+This document will guide you through the installation process and the first steps to get started with Thyra.
 
-* Thyra installation : obtain the Thyra binary corresponding to your operating system (OS), rename it and make it executable.
-* DNS configuration : install and configure your DNS to resolve the massa top level domain (*.massa) where Thyra runs.
 
-The following installation guide allows you to perform these steps automatically using a simple script to run on your terminal.
-
-Now, let's move on to your OS section:
+> **_TROUBLESHOOTING:_** If you have trouble following this procedure, feel free to [open a question](https://github.com/massalabs/thyra/issues/new) describing your problem.
 
-* [Linux](#linux)
-* [Windows](#windows)
-* [MacOS](#macos)
+## Instructions by Devices
 
+By following these instructions, you will: 
+* Install Thyra to access and perform actions on the blockchain. 
+* Install Thyra application to interact and use Thyra and its plugins very easily.
 
-### MacOS
+### Desktop
 
-Simply use copy/paste the cmd line below in your terminal. The installation process will start and a success message will be displayed once done.
+#### MacOS
 
+1. Go on [this page](https://.com/massalabs/thyra/releases/latest) and download the installer that corresponds to your MacOS configuration
+* If you have M1 chip or more, chose "thyra-installer_darwin_arm64" installer
+* Otherwise, chose "thyra-installer_darwin_amd64" installer
+2. Once downloaded, double clik on the file
+> **_NOTE:_** You may need to authorise the download of this file as your Mac won't recognise it.
```

Comment:
I think we should add more details on how to run the installer if MacOS security is bothering the user.
```suggestion
> **_NOTE:_** You may need to authorise the download of this file as your Mac won't recognise it. You can authorise the installer to run by right clicking on the installer and click on `Open` from Finder. A dialog box will open asking you if you are sure, press the `Open` button and the installer should start.
```